### ISSUE TITLE
Pygments - scrollbar

### ISSUE THREAD

**Issue by dazinator** [alexurquhart/hugo-geo#17]
When using pygments, there is no horizontal scrollbar when is wider than the page:

![image](https://cloud.githubusercontent.com/assets/3176632/21907882/5f7cb4cc-d909-11e6-9bd4-ef9f6a6de1e3.png)

For a live example, see: http://localhost:1313/blog/2016/07/25/aspnet-core-taghelpers---a-better-addtaghelper-type-resolver/

I haven't tried the alternative (highlight.js) so not sure how horizontal scrollbars are handled in that scenario.

**Comment by dazinator** [comment]
Manged to solve this by creating a custom footer, with this added to the bottom:
```
 <script>
  $(document).ready(function () {
	$( ".highlighttable" ).wrap("<div class='table-responsive'></div>");
});
  </script>
```

Basically, `pygments` outputs a code block as a table like this:

```
<table class="highlighttable"><tbody><tr><td><div class="linenodiv" style="background-color: #f0f0f0; padding-right: 10px"><pre style="line-height: 125%"></pre></div></td><td class="code"><div class="highlight" style="background: #f8f8f8"><pre style="line-height: 125%">Some Code
</pre></div>
</td></tr></tbody></table>
```

Because this is a bootstrap responsive theme, tables have to be wrapped in a div with the `table-responsive` class.

I'll leave this open in case you want to include this functionality in the next version!

**Comment by alexurquhart** [comment]
I don't use pygments on my site so I never would have seen this. Thanks for providing a fix, I included it in d54e167

**Comment by dazinator** [comment]
Nice one, cheers.
